cukinia/common_security_tests.d/sudo.conf.j2

```diff
@@ -35,7 +35,7 @@ as "SEAPATH-00153 - rules are not defined by negation" \
 CMD_WITHOUT_ARGS=$(
```

Comment:
In your commit message:
"User the value of ansible_user" -> "Use the value of ansible_user"